packages/big-design/src/components/Select/styled.tsx

```diff
@@ -19,6 +20,13 @@ export const StyledDropdownIcon = styled(ArrowDropDownIcon)<{ disabled?: boolean
     `}
 `;
 
+export const StyledListGroupHeader = styled(StyledGroupHeader)`
+  font-family: 'Source Sans Pro SemiBold', 'Helvetica Neue', Arial, sans-serif;
+  font-size: ${({ theme }) => theme.helpers.remCalc(14)};
+  margin-top: 1rem;
+  color: ${({ theme }) => theme.colors.secondary50};
```

Comment:
This might be a question for design, but should we unify all these stylings for both `Dropdown` and `Select` group headers?

Reply:
I'll confer w/design and ask for clarification. Should that be a separate ticket for `Dropdown`?

Reply:
No, we should probably tackle that within this one.